shared/output/gapic/gems/my_plugin/lib/google/gapic/generators/my_plugin_generator.rb

```diff
@@ -38,22 +38,6 @@ def initialize api
           # Configure these helper method to be used by the generator
           # use_helpers! :helper_method1, :helper_method2, :helper_method3
         end
-
-        # Generates all the files for the API.
-        #
-        # @return [Array<
-        #   Google::Protobuf::Compiler::CodeGeneratorResponse::File>]
-        #   The files that were generated for the API.
-        def generate
-          files = []
```

Comment:
For cloud we have this method and call `super`.

Reply:
Added it back as a stub.